definitions/npm/jspdf_v2.x.x/flow_v0.83.x-/jspdf_v2.x.x.js

```diff
@@ -0,0 +1,116 @@
+       */
+      options?: {|
+        /**
+         * The alignment of the text, possible values: left, center, right, justify.
+         */
+        align?: string,
+        /**
+         * Sets text baseline used when drawing the text, possible values: alphabetic, ideographic, bottom, top, middle, hanging
+         */
+        baseline?: string,
+        /**
+         * Rotate the text clockwise or counterclockwise. Expects the angle in degree.
+         */
+        angle?: number | Matrix,
+        /**
+         * Direction of the rotation. 0 = clockwise, 1 = counterclockwise.
+         */
+        rotationDirection?: number,
+        /**
+         * The space between each letter.
+         */
+        charSpace?: number,
+        /**
+         * The lineheight of each line.
+         */
+        lineHeightFactor?: number,
+        /**
+         * Flags for to8bitStream.
+         */
+        flags?: {|
+          /**
+           * Don't add BOM to Unicode-text.
+           */
+          noBOM?: boolean,
+          /**
+           * Autoencode the Text.
+           */
+          autoencode?: boolean,
+        |},
+        /**
+         * Split the text by given width, 0 = no split.
+         */
+        maxWidth?: number,
+        /**
+         * Set how the text should be rendered, possible values: fill, stroke, fillThenStroke, invisible, fillAndAddForClipping, strokeAndAddPathForClipping, fillThenStrokeAndAddToPathForClipping, addToPathForClipping.
+         */
+        renderingMode?: string,
+        /**
+         * Option for the BidiEngine
+         */
+        isInputVisual?: boolean,
+        /**
+         * Option for the BidiEngine
+         */
+        isOutputVisual?: boolean,
+        /**
+         * Option for the BidiEngine
+         */
+        isInputRtl?: boolean,
+        /**
+         * Option for the BidiEngine
+         */
+        isOutputRtl?: boolean,
+        /**
+         * Option for the BidiEngine
+         */
+        isSymmetricSwapping?: boolean,
+      |},
+      /**
+       * If transform is a number the text will be rotated by this value around the anchor set by x and y.
+       *
+       * If it is a Matrix, this matrix gets directly applied to the text, which allows shearing effects etc.; the x and y offsets are then applied AFTER the coordinate system has been established by this matrix. This means passing a rotation matrix that is equivalent to some rotation angle will in general yield a DIFFERENT result. A matrix is only allowed in "advanced" API mode.
+       */
+      transform: number | Matrix,
+    ): void;
+    save(fileName?: string): void;
+    // With jspdf-autotable plugin
+    autoTable(options: {|
+      head: Array<Array<string>>,
+      body: Array<Array<string>>,
+      startY: number,
+    |}): void;
+    lastAutoTable: {|
+      finalY: number,
+    |};
+    // End with jspdf-autotable plugin
+  }
+
+  declare module.exports: {|
```

Comment:
Seems like it also expose a default export.  

Also prefer declaring the modules with ES module syntax, I just tested it (again) and you can commonjs `require()` a module declared as ES module just fine.